wbooks/src/app/index.tsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { StatusBar } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
 import LibraryScreen from '@screens/Library';
+import BookDetailScreen from '@screens/BookDetail';
+import { ROUTES } from '@constants/routes';
+
+const LibraryStack = createStackNavigator();
 
 const App = () => {
   return (
-    <>
-      <StatusBar barStyle="dark-content" />
-      <LibraryScreen />
-    </>
+    <NavigationContainer>
+      <LibraryStack.Navigator initialRouteName={ROUTES.library}>
+        <LibraryStack.Screen name={ROUTES.library} component={LibraryScreen} />
+        <LibraryStack.Screen name={ROUTES.bookDetail} component={BookDetailScreen} />
+      </LibraryStack.Navigator>
```

Comment:
Move this to `AppNavigator` component (create it):
```
<NavigationContainer>
  <StatusBar barStyle="dark-content" />
  <AppNavigator />
</NavigationContainer>
```
We use `AppNavigator` component for all the navigation logic and stacks/tabs navigators